Compiler errors on Windows x86

When compiling the v3.27.1 source code on Windows x86, an error occurs:

```console
C:\Users\runneradmin\AppData\Local\.xmake\cache\packages\2606\f\freerdp\3.27.1\source\libfreerdp\core\update.c(3655): error C2440: 'function': cannot convert from 'BOOL (__cdecl *)(PINIT_ONCE,PVOID,PVOID *)' to 'PINIT_ONCE_FN'
```

This is because the calling convention for a recently added callback function differs from that of `PINIT_ONCE_FN`.

--- a/libfreerdp/core/update.c
+++ b/libfreerdp/core/update.c
@@ -54,9 +54,9 @@
 static INIT_ONCE stats_names_once = INIT_ONCE_STATIC_INIT;
 static char stats_names[RDP_STATS_COUNT][bufferlen];
 
-static BOOL stats_names_generate(WINPR_ATTR_UNUSED PINIT_ONCE InitOnce,
-                                 WINPR_ATTR_UNUSED PVOID Parameter,
-                                 WINPR_ATTR_UNUSED PVOID* Context)
+static BOOL CALLBACK stats_names_generate(WINPR_ATTR_UNUSED PINIT_ONCE InitOnce,
+                                          WINPR_ATTR_UNUSED PVOID Parameter,
+                                          WINPR_ATTR_UNUSED PVOID* Context)
 {
 	for (size_t index = 0; index < RDP_STATS_COUNT; index++)
 	{

